src/components/NewBeer.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+function NewBeer() {
+	const [formData, setFormData] = useState({
+		name: '',
+		tagline: '',
+		description: '',
+		first_brewed: '',
+		brewers_tips: '',
+		attenuation_level: 0,
+		contributed_by: '',
+	});
+
+	const navigate = useNavigate();
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+
```

Comment:
I would add that one, so you avoid sending information that won't be valid in the backend either way

```suggestion
  if (!formData.name) {
    setErrorMessage('We need at least the name to create a new beer')
    return
  }
```